src/Util/Util.js

```diff
@@ -566,25 +566,72 @@ const Util = {
   },
 
   /**
+   * Underscore.js 1.8.3. For more information,
+   * see http://underscorejs.org/docs/underscore.html#section-82
    * @param {Function} func A callback function to be called
    * @param {Number} wait How long to wait
-   * @returns {Function} A function, that, as long as it continues to be
-   * invoked, will not be triggered. The function will be called
-   * after it stops being called for N milliseconds.
+   * @param {Object} [options]
+   * @param {Object} [options.leading = false] Run function execution on the
```

Comment:
Is it possible enable/disable both leading and trailing at the same time? It might makes sense to combine these into one flag...